### ISSUE TITLE
[Bug] Potential Antimalware Scan Interface Bypass via PowerShell

### ISSUE THREAD

**Issue by MakoWish** [elastic/detection-rules#2662]
**Describe the bug**
The values containing a period `.` in this rule are being treated as individual values. For instance, the value `Bypass.AMSI` is being treated as two separate values `Bylass` and `AMSI`. Each of these values should be double-quoted to be queried as literal strings. 

**To Reproduce**
1. Enable rule
2. Create arbitrary PowerShell PS1 script with any simple task such as:
    ```
    Set-ExecutionPolicy Bypass
    Write-Output "Hello World"
    ```
4. Execute this script
5. See false detection due to `Bypass` in the ExecutionPolicy

**Expected behavior**
Since the detection rule is looking for `Bypass.AMSI`, it should not trigger for just having the string `Bypass` in the script block text.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: All

**Comment by w0rk3r** [comment]
Hey @MakoWish, thanks again for contributing! I think I fixed this one on https://github.com/elastic/detection-rules/pull/2614, can you confirm if the fix works on your end? This will be included in our next rule release

**Comment by w0rk3r** [comment]
@MakoWish, you're right. That was an overlook on my end, I've pushed a new [PR](https://github.com/elastic/detection-rules/pull/2663) to fix the remaining keywords. Thanks, again!

**Comment by w0rk3r** [comment]
Ooh, sorry about that, lol. looking forward to your PRs in the future, and will ask next time if you want to do the PR 😅

**Comment by MakoWish** [comment]
All good! Thank you! 👍
